model/internal/otlp_wrapper.go

```diff
@@ -38,6 +39,58 @@ func MetricsFromOtlp(req *otlpcollectormetrics.ExportMetricsServiceRequest) Metr
 	return MetricsWrapper{req: req}
 }
 
+func intHistogramToHistogram(src *otlpmetrics.Metric_IntHistogram) *otlpmetrics.Metric_Histogram {
```

Comment:
nit: usually I prefer public funcs at the top :)

Reply:
Moved this function to the bottom.